deploy/role.yaml

```diff
@@ -9,9 +9,15 @@ rules:
 - apiGroups: [apps]
   resources: [deployments, daemonsets]
   verbs: [create, get, list, patch, delete, update, watch]
+- apiGroups: [admissionregistration.k8s.io]
+  resources: [mutatingwebhookconfigurations, validatingwebhookconfigurations]
+  verbs: [get, update, create]
```

Comment:
Role additions will also needed a change for bundle generation.
for bundle generation kubebuilder is used for generating the CSV and role yaml files. More details are available here: https://github.com/vmware/antrea-operator-for-kubernetes/pull/25

In theory we could first merge that PR and then update this one in order to avoid dealing with conflicts.
This is only for OLM adoption.

Reply:
> Role additions will also needed a change for bundle generation.
> for bundle generation kubebuilder is used for generating the CSV and role yaml files. More details are available here: #25
> 
> In theory we could first merge that PR and then update this one in order to avoid dealing with conflicts.
> This is only for OLM adoption.

It's ok, I will rebase and fix conflicts after that PR merged.

Reply:
Awesome